packages/nest/README.md

```diff
@@ -25,11 +25,46 @@
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
 for feature flagging that works with your favorite feature flag management tool.
 
+<!-- x-hide-in-docs-end -->
+
 🧪 This SDK is experimental.
 
-#### Here's a basic example of how to use the OpenFeature NestJS API with `InMemoryProvider`.
+## Overview
+
+The OpenFeature NestJS SDK is a package that provides a NestJS wrapper for the [OpenFeature Server SDK](https://openfeature.dev/docs/reference/technologies/server/javascript/).
+
+It's main capabilities are:
+
+- Provide a NestJS global module to simplify OpenFeature configuration and usage within NestJS;
+- Supply custom parameter decorators (for controllers), that inject observables with feature flags resolution details;
```

Comment:
```suggestion
- Injecting feature flags into directly controller route handlers
```

Reply:
I think, this is more about the feature, so the addition that it is using Observables is an implementation detail.